cpp/include/cudf/column/column_view.hpp

```diff
@@ -375,6 +379,44 @@ class column_view : public detail::column_view_base {
    */
   auto child_end() const noexcept { return _children.cend(); }
 
+  /**
+   * @brief Construct a column view from a device_span<T>.
+   *
+   * Only numeric and chrono types are supported.
+   *
+   * @tparam T The device_span data type.
+   * @param data The device_span containing the column elements.
+   */
+  template <typename T, std::enable_if_t<cudf::is_numeric<T>() or cudf::is_chrono<T>()>* = nullptr>
+  [[nodiscard]] column_view(device_span<T> data)
```

Comment:
should probably verify that the `data.size() < std::numeric_limits<size_type>::max()`.

Reply:
Good idea. Resolved in c575f579cf29928541bf31d804789b0ad624e247.